### ISSUE TITLE
EXPID, EXPDSC, and EXPSRC in HISTORY.rc

### ISSUE THREAD

**Issue by gmao-rreichle** [GEOS-ESM/GEOSldas#469]
@weiyuan-jiang, @mathomp4:
The [HISTORY.rc template in GEOSldas](https://github.com/GEOS-ESM/GEOSldas/blob/21aab7f5fdb5b1112c43e3248be9c148f50ec6de/src/Applications/LDAS_App/GEOSldas_HIST.rc#L9) includes the following resource parameters near the beginning of the file:

```
EXPID:  GEOSldas_expid
EXPDSC: GEOSldas_output
EXPSRC: GEOSldas
```
Does the entry for EXPID matter for anything?  What about EXPDSC and EXPSRC?  I'm guessing these strings somehow end up in the metadata of nc4 HISTORY output but are ignored for binary HISTORY output.

I'm asking because it was confusing to a user, who wanted to know if they have to change the EXPID entry when re-using the HISTORY file for another simulation. 

If the rc parameters in question are irrelevant other than for nc4 metadata, I'll create a PR that adds a comment along those lines to `GEOSldas_HIST.rc`.

**Comment by gmao-rreichle** [comment]
Many thanks, @weiyuan-jiang, for the input.  I faintly remember that this was changed to a constant string a while back.  But the following is still unclear to me:
- Could we simply remove/comment-out these resource parameters without breaking the simulation? 
- If the resource parameters are specified, what becomes of them?  (e.g., strings added to metadata in nc4 file?)
- Is there documentation (other than the MAPL source code) that explains the above?  (@mathomp4 mentioned that Ben has some rudimentary documentation of MAPL HISTORY.  From what I can tell, though, the [MAPL Wiki](https://github.com/GEOS-ESM/MAPL/wiki/MAPL-History-Component) doesn't answer my questions.  I might have missed something.)

If we the resource parameters do absolutely nothing, we should delete them.

If we can comment them out without breaking anything, we should comment them out.

If specifying these resource parameters can do something that may be useful, we should add comments in GEOSldas_HIST.rc that explains what the parameters are good for, so users can make an educated decision whether they want to specify something and what it should be.

In summary, unless we get rid of the parameters or give users some help (a brief comment or pointer to documentation), we'll have to keep answering questions from users.
